server/src/tasks/sqs/bridge.js

```diff
@@ -96,6 +97,8 @@ const mintDeposited = async (account, { depositId, bridgeType, tokenAddress, rec
     })
 
     if (receipt.status) {
+      const { blockNumber } = receipt
+      await job.set('data.transactionBody', { ...lodash.get(job.data, 'transactionBody', {}), status: receipt.status ? 'confirmed' : 'failed', blockNumber }).save()
```

Comment:
@leonprou  This `receipt.status ? 'confirmed' : 'failed'` seems to be redundant cuz we know `receipt.status` is true

Reply:
the tx may failed. so the receipt will be false